addons/misra.py

```diff
@@ -1950,7 +1950,7 @@ def misra_15_6(self, rawTokens):
                 if token.str.startswith('//') or token.str.startswith('/*'):
                     continue
                 state = 0
-                if token.str != '{':
+                if token.str != '{' and '#' not in token.str:
```

Comment:
what is the offending token.str? would this rewritten condition work?

    if token.str not in ('{', '#'):

Reply:
Thanks, yes, it will work.